lib/js/components/Banner/Banner.vue

```diff
@@ -189,24 +183,45 @@
 		margin-top: $space-xxxxs;
 	}
 
-	&__text {
+	&__textWrapper {
+		flex: 1;
+		display: flex;
+		flex-direction: column;
+		justify-content: space-between;
+	}
+
+	&__titleWrapper {
 		display: flex;
 		flex-direction: column;
 		justify-content: center;
 		flex-grow: 1;
 		padding: $space-xxxxs 0;
 	}
 
-	&__iconContainer {
-		@include iconContainerVerticalStyles;
+	&__rightWrapper {
+		display: flex;
```

Comment:
becuase of `flex`, button is growing when text goes into multiple lines
![Zaznaczenie_917](https://user-images.githubusercontent.com/2725894/193032159-0632f0fe-ac35-4002-9d63-c62bb41bd574.png)

Reply:
fiexed, so wrapper are required :D